### ISSUE TITLE
When given a file of URLs, continue even if one is bad

### ISSUE THREAD

**Issue by bagder** [curl/trurl#76]
This case should output four URLs - there is one bad one in the middle.

~~~
$ cat urls.txt 
localhost
haxx.se
ftp://ex ample.com
https://curl.se
curl.haxx.se
$ trurl -f urls.txt
http://localhost/
http://haxx.se/
trurl note: Bad hostname [ftp://ex ample.com]
trurl error: not enough input for a URL
trurl error: Try trurl -h for help

**Comment by bharsaklemukesh975** [comment]
can I work on this? I would like to start my first open source contribution with this .
Thanks

**Comment by jay** [comment]
@havbon has already submitted a fix in #84

**Comment by jay** [comment]
If a user passes in 5 lines I would think they expect to get 5 lines back, so what about if bad URLs just result in blank lines
